### PR TITLE
[MDS-5860] added detailed message to network error toasts

### PR DESCRIPTION
**Pull request by matbusby-fw**
## Objective 

[MDS-5860](https://bcmines.atlassian.net/browse/MDS-5860)

Add the detailed error message to the toast message that pops up on errors returned from the API

### CODE REVIEW

File: services/common/src/redux/customAxios.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { notification, Button } from "antd";
 import * as String from "@mds/common/constants/strings";
 import React from "react";
 import * as API from "@mds/common/constants/API";
-import { ENVIRONMENT } from "@mds/common";
+import { ENVIRONMENT } from "@mds/common/constants";
```
Comment (taraepp) [inline services/common/src/redux/customAxios.js:6]: it's a bit funny to me to see your one-man mission to make imports better. 😂 
Keep fighting the good fight!

**Review by taraepp — APPROVED** [review]

**Review by henryoforeh-dev — APPROVED** [review]

File: services/common/src/redux/customAxios.js (hunk 1 above)
Comment (matbusby-fw) [reply services/common/src/redux/customAxios.js:6]: Haha, my IDE complains about it, and I hate seeing red lines

**Review by isuru-aot — APPROVED** [review]

File: services/common/src/redux/customAxios.js (hunk 1 above)
Comment (taraepp) [reply services/common/src/redux/customAxios.js:6]: ahhhh mine doesn't, so I assumed it was preference. But mine did start complaining about unused imports in python recently, which I have been enjoying. 🪓

Comment (matbusby-fw) [reply services/common/src/redux/customAxios.js:6]: It is satisfying to do these little cleanups.  With the common repo there's also the fact that there are many variables with identical names in different folders (legacy from before the common repo).  It all seems to mostly work, but it has the potential to break without more specific imports.

**Pull request merged**